src/HTMLTIDY.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="HTMLTIDY" name="HTML Tidy License">
```

Comment:
@johnmhoran, can you update this `license` tag to include `listVersionAdded="3.11"` ?

Reply:
Will do @swinslow -- thank you.  👍